### PR TITLE
use op indexer for op deposits & withdrawals

### PR DESCRIPTION
**Pull request by lukasrosario**
**What changed? Why?**
fetch op deposits / withdrawals from our deployment of the op indexer

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: apps/bridge/src/contexts/OFACContext.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export async function fetchIsAllowed(address?: `0x${string}`): Promise<{ result:
     return { result: false };
   }
 
-  const response = await fetch(publicRuntimeConfig.complianceApiURL, {
+  const response = await fetch(publicRuntimeConfig.bridgeApiURL, {
```
Comment (zencephalon) [inline apps/bridge/src/contexts/OFACContext.tsx:27]: Just to make sure I understand this correctly, does the compliance endpoint run through this same API then?

File: apps/bridge/src/utils/hooks/usePrepareFinalizeWithdrawal.ts [hunk 2]
```diff
@@ -4,44 +4,52 @@ import { WithdrawalMessage } from 'apps/bridge/src/types/WithdrawalMessage';
...
-        {
-          nonce: withdrawalForTx.nonce,
-          sender: withdrawalForTx.sender,
-          target: withdrawalForTx.target,
-          value: withdrawalForTx.value,
-          gasLimit: withdrawalForTx.gasLimit,
-          data: withdrawalForTx.data,
-        },
-      ]
+          {
```
Comment (zencephalon) [inline apps/bridge/src/utils/hooks/usePrepareFinalizeWithdrawal.ts:34]: Is there any reason not to just pass in `withdrawalForTx` here? It seems like we just read all the same keys into a new object? If it needs to be a new object for some reason could we do `{...withdrawalForTx}`?

File: apps/bridge/src/contexts/OFACContext.tsx (hunk 1 above)
Comment (lukasrosario) [reply apps/bridge/src/contexts/OFACContext.tsx:27]: yup, just updating the name because it's doing more than just compliance now

Comment (zencephalon) [reply apps/bridge/src/contexts/OFACContext.tsx:27]: Ok, sweet

File: apps/bridge/src/utils/hooks/usePrepareFinalizeWithdrawal.ts (hunk 2 above)
Comment (lukasrosario) [reply apps/bridge/src/utils/hooks/usePrepareFinalizeWithdrawal.ts:34]: hm yeah not sure why this was set up like this. can follow up & clean up

**Pull request merged**